### DISCUSSION TITLE
Getting a 404 after upgrading dependencies

### DISCUSSION THREAD

**Discussion by nerdroid23** [filamentphp/filament#3766] [Help] [answered]
Currently on version `v2.12.14`. After running `composer update` and then `php artisan filament:upgrade`, I get a 404. My site does not have any other routes and when I run `php artisan route:list` I do see the login route but it returns a 404.

<img width="921" alt="image" src="https://user-images.githubusercontent.com/21242932/187567301-7166079a-dc1b-4b18-9ce7-a8e12b788ce2.png">

Has anyone encountered this?

**Comment by danharrin** [comment, marked as answer]
Please check your service providers. In `register()`, do you have anything Filament-related? That should all be in `boot()`, as per the Laravel docs.

**Reply by nerdroid23** [reply]
Ah, thanks. That was indeed the issue. I was not aware of these changes.

**Reply by danharrin** [reply]
https://laravel.com/docs/9.x/providers#the-register-method
